wrappers/core18_test.go

```diff
@@ -165,6 +169,15 @@ WantedBy=snapd.service
 	}, {
 		filepath.Join(dirs.SnapDBusSessionPolicyDir, "snapd.session-services.conf"),
 		"<busconfig/>",
+	}, {
```

Comment:
we might want also a new or some relevant preexisting spread core tests to check the files are present now